src/util/dev/server.ts

```diff
@@ -388,6 +388,22 @@ export default class DevServer {
       }
     }
 
+    if (Array.isArray(config.builds)) {
+      // `@now/static-build` needs to be the last builder
+      // since it might catch all other requests
+      config.builds = config.builds.sort((buildA, buildB) => {
```

Comment:
`Array#sort()` modifies the array in-place, so no need to re-assign.

```suggestion
      config.builds.sort((buildA, buildB) => {
```